src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlService.java

```diff
@@ -568,12 +569,21 @@ default boolean hasCaseEventAccess(String eventId,
                                        List<CaseEventDefinition> caseEventDefinitions,
                                        Set<AccessProfile> accessProfiles,
                                        Predicate<AccessControlList> criteria) {
-        for (CaseEventDefinition caseEvent : caseEventDefinitions) {
-            if (caseEvent.getId().equals(eventId)
-                && hasAccessControlList(accessProfiles, criteria, caseEvent.getAccessControlLists())) {
-                return true;
-            }
+        Optional<CaseEventDefinition> matchedEvent = caseEventDefinitions.stream()
+                .filter(caseField -> caseField.getId().equals(eventId))
```

Comment:
caseField name should be changed to caseEvent

Reply:
Done ✅